app/uk/gov/hmrc/traderservices/controllers/ContactFieldHelper.scala

```diff
@@ -35,4 +35,10 @@ object ContactFieldHelper {
           case _: NumberParseException => Invalid(ValidationError(errorMessage))
         }
     }
+
+  def normaliseNumber(phoneNum: String): String =
+    if (phoneNum.startsWith("+44")) phoneNum.replace("+44", "0").replaceAll("\\s", "")
+    else if (phoneNum.startsWith("44")) phoneNum.replace("44", "0").replaceAll("\\s", "")
```

Comment:
we can use `.replaceAll("\\s", "")` at the end

Reply:
I'm happy that this story meets the AT now & have approved - @arturopala regarding this comment did you want to just have one .replaceAll("\\s", "") at the very end instead of at the end of each line? 

Asides from this I think this PR is good to be merged